modules/database/src/ioc/registry/registryIocRegister.c

```diff
@@ -21,28 +21,44 @@ static const iocshArg * const registryXxxFindArgs[1] = {&registryXxxFindArg0};
 
 /* registryRecordTypeFind */
 static const iocshFuncDef registryRecordTypeFindFuncDef = {
-    "registryRecordTypeFind",1,registryXxxFindArgs};
+    "registryRecordTypeFind",
+    1,
+    registryXxxFindArgs,
+    "Prints the registry address of the record type given as first argument.\n",
+};
 static void registryRecordTypeFindCallFunc(const iocshArgBuf *args) {
     printf("%p\n", (void*) registryRecordTypeFind(args[0].sval));
 }
 
 /* registryDeviceSupportFind */
 static const iocshFuncDef registryDeviceSupportFindFuncDef = {
-    "registryDeviceSupportFind",1,registryXxxFindArgs};
+    "registryDeviceSupportFind",
+    1,
+    registryXxxFindArgs,
+    "Prints the registry address of the device support given as first argument.\n",
```

Comment:
Please add an example. It's not clear in what format the argument should be, i.e. prefixed with `dev` or not, capitalization ...